packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -345,11 +353,11 @@ where
 
         let version_id = VersionId::new(Uuid::new_v4());
         self.insert_version_id(version_id).await?;
-        self.insert_uri(&uri, version_id).await?;
-
-        self.insert_with_id(version_id, database_type, owned_by_id, updated_by_id)
+        self.insert_uri(&uri, version_id, owned_by_id, updated_by_id)
             .await?;
 
+        self.insert_with_id(version_id, database_type).await?;
```

Comment:
I suppose the SQL `function`s are yet to be made, are they going to be in a follow up PR?

Reply:
Yes, I'm currently cleaning up my branch and will open a PR with the functions in a couple minutes 🚀